packages/usa-input-prefix-suffix/src/usa-input-prefix.twig

```diff
@@ -22,7 +22,7 @@
   <div class="usa-input-group usa-input-group--error">
     <div class="usa-input-prefix" aria-hidden="true">
       <svg aria-hidden="true" role="img" focusable="false" class="usa-icon">
-        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="./img/sprite.svg#credit_card"></use>
+        <use xmlns:xlink="http://www.w3.org/1999/xlink" href="./img/sprite.svg#credit_card"></use>
```

Comment:
**question:** Is the use of `xmlns:xlink` required here? Or is it safe to use since we removed `xlink:href`?

Reply:
Resolved in [01b1427](https://github.com/uswds/uswds/pull/6165/commits/01b1427f7e257117b6ace5b434929bc083092509). From what I've found, it isn't necessary for us to keep the `xmlns:xlink` attribute anymore, so they have been removed.